metron-deployment/packaging/docker/ansible-docker/Dockerfile

```diff
@@ -21,10 +21,10 @@ RUN yum install -y tar
 RUN yum install -y wget
 RUN yum groupinstall -y "Development tools"
 RUN yum install -y zlib-dev openssl-devel sqlite-devel bzip2-devel libffi-devel
-RUN wget https://www.python.org/ftp/python/2.7.10/Python-2.7.10.tgz -O /usr/src/Python-2.7.10.tgz
+RUN wget https://www.python.org/ftp/python/2.7.12/Python-2.7.12.tgz -O /usr/src/Python-2.7.12.tgz
```

Comment:
I thought we were pinning to 2.7.11 generally.  I know it's a bit pedantic, as the differences are minimal, but perhaps we should just stay with a single version across all of our builds?